pkg/translator/prometheusremotewrite/otlp_to_openmetrics_metadata_test.go

```diff
@@ -237,6 +237,70 @@ func TestOtelMetricsToMetadata(t *testing.T) {
 	}
 }
 
+func TestOtelMetricsToMetadataNamespace(t *testing.T) {
```

Comment:
What do you think about adding a namespace to the test above this one, instead of creating a whole new test? You can add just one case with a non-empty namespace, and keep the rest as it is 🤔